### DISCUSSION TITLE
Code always opens two tabs | --ozone-platform-hint=wayland --disable-gpu

### DISCUSSION THREAD

**Discussion by root9191** [prasanthrangan/hyprdots#1564] [Q&A]
Hi all, 
when i open Code which got installed with this Script i always have this two tabs:
![grafik](https://github.com/prasanthrangan/hyprdots/assets/47309435/868e751f-7293-48e2-a92e-4f98769539d1)
Why does that happen? Can i fix that somehow?

Is there also any kind of documentation how to use your dots like in your Video?
For example how did you got Discord to change to actual theme? 
Which app are you using for listening Music and what is this program?
![grafik](https://github.com/prasanthrangan/hyprdots/assets/47309435/56df485f-189e-4f57-86dd-1a6bec9442fc)

**Comment by AKA-Akhil** [comment]
The discord that is installed by the script is actually Webcord. WebCord is a discord client that brings discord web to desktop. 
if you want you can install [discord](https://flathub.org/apps/com.discordapp.Discord) using [flathub](https://flathub.org/en) and mod it using [vencord](https://vencord.dev/) or [BetterDiscord](https://betterdiscord.app/) which allows you to add themes and plugins.

As for the music player you can download [spotify](https://flathub.org/apps/com.spotify.Client) from [flathub](https://flathub.org/en)
then use [spicetify](https://spicetify.app/) which is used to customize spotify.

**Reply by root9191** [reply]
Thank you, i will take a look at it in the evening.
Do you also know why VS Code opens this two tabs with something like --disable-gpu?

**Reply by root9191** [reply]
Can you also give some details about that color picker or whatever this should be 😅 
![image](https://github.com/prasanthrangan/hyprdots/assets/47309435/ee830846-0df8-420b-b572-b592bfb5e3ec)
Do you also know for what Electron got installed? Can't i also use this for kind of theming apps?

**Comment by AKHIL-DyC** [comment]
> Hi all, when i open Code which got installed with this Script i always have this two tabs: ![grafik](https://private-user-images.githubusercontent.com/47309435/336417901-868e751f-7293-48e2-a92e-4f98769539d1.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTc1MzA3NDAsIm5iZiI6MTcxNzUzMDQ0MCwicGF0aCI6Ii80NzMwOTQzNS8zMzY0MTc5MDEtODY4ZTc1MWYtNzI5My00OGUyLWE5MmUtNGY5ODc2OTUzOWQxLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA2MDQlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNjA0VDE5NDcyMFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTA2NGYxMjQ1N2FmYmMwMTBiMzljNGFjYzAzYTA5YzI4NDRiMjNmMDk3ODlhNGQwYjA4ZTdhZWJjZDA1MGFlN2MmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.gIT1T6YKMAg0C_q78tOdTYcp_zFrivrDPAxnqAIxyrU) Why does that happen? Can i fix that somehow?
> 
> Is there also any kind of documentation how to use your dots like in your Video? For example how did you got Discord to change to actual theme? Which app are you using for listening Music and what is this program? ![grafik](https://private-user-images.githubusercontent.com/47309435/336418392-56df485f-189e-4f57-86dd-1a6bec9442fc.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTc1MzA3NDAsIm5iZiI6MTcxNzUzMDQ0MCwicGF0aCI6Ii80NzMwOTQzNS8zMzY0MTgzOTItNTZkZjQ4NWYtMTg5ZS00ZjU3LTg2ZGQtMWE2YmVjOTQ0MmZjLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA2MDQlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNjA0VDE5NDcyMFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTY1ZWNhODJjMTMzMzAxZGJhZmYzOTkyYzVhOWEwOGJjNjhiNTkwNTRiNmJhYmMyMDQ3MzU2MThkYjAyNmJmY2EmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.ak3cejya3Oo7_GejG2kYZbbU76cWCqIFT4QzKC-jx4o)

take a look at https://github.com/prasanthrangan/hyprdots/issues/1315

**Reply by root9191** [reply]
Thank you that helped a lot, now also the flicker in Code is gone :)
So i understand also a bit better what Electron does. 
Can i customize the Font Colors in Code too?

**Reply by AKA-Akhil** [reply]
Well If you are referring to the overall theme, you can just download one form the extensions tab.

If you dont want a theme but want to change the colors then you can just edit the json file by doing this
1)Press CTRL+SHIFT+P
2)Type Open Settings and click on it (NOT THE DEFALT ONE)
3)Make whatever changes you want
